wranglertools/import_data.py

```diff
@@ -1325,9 +1330,21 @@ def cabin_cross_check(connection, patchall, update, infile, remote):
                 sys.exit(1)
 
 
-def get_collections(connection):
+def get_profiles(connection):
+    return ff_utils.get_metadata("/profiles/", key=connection.key, add_on="frame=object")
+
+
+def get_attachment_fields(profiles):
+    attach_field = []
+    for _, profile in profiles.items():
```

Comment:
More style preference than anything, but instead of creating an unused silent variable why not:
for profile in profiles.values():